backend/README.md

```diff
@@ -0,0 +1,103 @@
+# Setup Guide
+
+## Step 1: Set up the server
+
+1. **Open Command Prompt:**
+   Open the command prompt on your machine.
+
+2. **Navigate to the project directory:**
+   Go to the server folder of your project:
+
+   ```bash
+   cd <your-project-path>/server
+
```

Comment:
Here it should be `<your-project-path>/backend`, isn't it? since we do not have any folder name server

Reply:
I changed `cd <your-project-path>/server` to `<your-project-path>/backend`